pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "plc4x"
+version = "0.1.0"
+description = ""
+authors = ["ottlukas <ott.lukas.14@gmail.com>"]
```

Comment:
This should better be "Apache PLC4X PMC"

Reply:
yes ... strange in PyCharm it looks like this:
![image](https://user-images.githubusercontent.com/19743877/104427110-2719a180-5583-11eb-93ae-5dfa0b1b60c4.png)
But when I push to repo it does not...any ideas how to change this (seems to be IDE related)